nwbinspector/checks/tables.py

```diff
@@ -111,7 +111,7 @@ def check_column_binary_capability(table: DynamicTable, nelems: int = 200):
             if np.asarray(column.data[0]).itemsize == 1:
                 continue  # already boolean, int8, or uint8
             try:
-                unique_values = np.unique(_subsample_data(data=column, nelems=nelems))
+                unique_values = np.unique(_cache_data_selection(data=column.data, selection=slice(nelems)))
```

Comment:
```suggestion
                unique_values = np.unique(_cache_data_selection(data=column.data, selection=slice(0, nelems)))
```
should this be `slice(0, nelems)`?